packages/exp/src/commands/upload/IOSUploader.js

```diff
@@ -0,0 +1,49 @@
+import prompt from '../../prompt';
+import log from '../../log';
+
+import BaseUploader from './BaseUploader';
+import { printFastlaneError, spawnAndCollectJSONOutputAsync } from './utils';
+
+export default class IOSUploader extends BaseUploader {
+  constructor(projectDir, options) {
+    super(projectDir, options, 'ios', 'iOS', 'ipa');
+  }
+
+  ensurePlatformOptionsAreCorrect() {}
```

Comment:
Maybe get rid of `throw new Error('Not implemented');` or even the whole function in the base class, so that you don't have to define an empty function here.